.changeset/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "bigcommerce/catalyst" }
+  ],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "privatePackages": {
+    "version": true,
+    "tag": true
+  },
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": []
```

Comment:
⚠️ There is a chance we might want to use this field for `apps/docs` and `packages/functional` so that we can keep them out of NPM, and also not have to worry about versioning them.

Reply:
Since `ignore` is meant to be used as a [temporary solution](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#ignore-array-of-packages), we'll just leave `packages/functional` with the `private` property in `package.json` set to `true`.